dlk/README.md

```diff
@@ -135,7 +135,7 @@ PYTHONPATH=python/dlk python setup.py test
 ```
 
 # Development
```

Comment:
Link to LM internal document is not desirable. I think it's better to remove this paragraph itself, since the "LeapMind python coding philosophy" is incomplete and should not be published.

Reply:
The change was just by editor conf, but I will remove whole sentence

Reply:
Changed